lib/fastlane/plugin/fueled/actions/generate_changelog.rb

```diff
@@ -118,13 +118,17 @@ def self.format_scope(scope, formatting_type)
         end
 
         scope.strip!
-        if is_linear_issue(scope)
+        if is_ticket_assigned(scope)
           formatted_scope = scope.upcase
+          base_url = ENV["TICKET_BASE_URL"]
```

Comment:
💯

Reply:
And same here I suppose, having this as part of the `available_options` will document this env var usage. And we could even default to the Linear base URL for easy back compat.